test/getrequest.test.js

```diff
@@ -0,0 +1,28 @@
+const axios = require('axios');
+const assert = require('assert');
+const fs = require('fs');
+const chai = require('chai');
+chai.use(require('chai-json-schema'));
+describe('GET method tests', () => {
+        let rawdata = fs.readFileSync('./test/resources/get_rs.json');
+        let expectedData = JSON.parse(rawdata);
+        let expectedSchema = JSON.parse(fs.readFileSync('./test/resources/get_rs.schema'));
+        it('Return 10 users from https://jsonplaceholder.typicode.com/', async function () {
+                await axios({
+                        method: 'GET',
+                        url: 'https://jsonplaceholder.typicode.com/users',
+                        params: {
+                                _limit: 10
```

Comment:
better to put value 10 to file or declare it with "let" for avoiding hardcoding

Reply:
fixed for all comments in bc0285d